global/meta/models.yml

```diff
@@ -2700,6 +2700,7 @@ poll:
       - analog
       - named
       - pseudoanonymous
+      - crypt
```

Comment:
I think I would favor "anonymous". If you disagree, I would at least write the full word ("cryptographic") to stay in lane with the other names.

Reply:
I have no preference. 

anonymous sounds like the poll is for anonymous users.

The English term for "geheime Wahl" is "secret ballot". So another option would be "secret".

Reply:
That is right, also we already have a similar "problem" with "pseudoanonymous". But "cryptographic" but be better suited under these circumstances.

Reply:
done